src/index.html

```diff
@@ -0,0 +1,11 @@
+<html>
+    <!DOCTYPE html>
```

Comment:
```suggestion
<!DOCTPE html>
<html>
```

The doctype has to be in the beginning of the file. [see here](https://developer.mozilla.org/en-US/docs/Web/HTML/Quirks_Mode_and_Standards_Mode#how_does_mozilla_determine_which_mode_to_use.3f)